pages/index.js

```diff
@@ -1,23 +1,25 @@
 import Head from "next/head";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
-import Cards from "../components/Cards";
+import Page from "../components/Page";
 
 // Importing stylesheets
 import globalStyles from "../styles/global";
 
 export default function Home() {
-    return (
-        <div className="container">
-            <Head>
-                <title>Evolution</title>
-                <link rel="icon" href="/favicon.ico" />
-            </Head>
-            <Header />
-            <Footer />
-            <style jsx global>
-                {globalStyles}
-            </style>
-        </div>
-    );
+  return (
+    <div className="container">
+      <Head>
+        <title>Evolution</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Page>
+        <Header />
+        <Footer />
+      </Page>
+      <style jsx global>
+        {globalStyles}
+      </style>
```

Comment:
I think you can delete this. It's inside the `Page` component.

Reply:
will this  affect the consistency of the styling for each page you create if removed?

Reply:
I think every page should use the `Page` component. That component should be the source of the consistency.

Reply:
You might be able to move into the `Head` component. I'm not sure though. I'm not familiar with this jsx global stuff. Usually the CSS goes in the head.

Reply:
Oh speaking of the `Head` component. I think the about page should also have a head component. You could probably even move that into the `Page` component. You could also probably get rid of the parent div in this component as well. Make `Page` the root node for this component.

Reply:
That would clean things up a lot. This component becomes.

```jsx
<Page>
   <Header />
    <Footer />
</Page>
```